doc/releases/changelog-dev.md

```diff
@@ -335,6 +350,12 @@
   4: ─────────────────╰RY(0.81)─╰SWAP─────────────────╰RY(0.06)─╰SWAP─────────────────┤  State
   ```
 
+<h3>Improvements</h3>
+
+<h4>Add new improvements here</h4>
+
+<h4>Pulse programming</h4>
```

Comment:
I'm neutral. For large features like this, it makes sense to me for the "headline" stuff to live in the new features section, and some of the supporting additions to live in the improvements.

I added these categories to improvements, but might not do it next time since I don't think it's worth the added work :laughing: